apps/files/src/components/FileDetails.vue

```diff
@@ -1,55 +1,46 @@
 <template>
-  <v-navigation-drawer
-  v-model="drawer"
-  class="grey lighten-2"
-  right
-  floating
-  permanent
-  style="width:100%"
-  >
-  <v-layout column>
-    <v-layout primary row>
-      <v-flex>
-        <v-icon color="white" class="pl-4" medium>folder</v-icon>
-      </v-flex>
-      <v-flex white--text align-self-center>
-        <span class="subheading"> {{ getTabName }} </span>
-      </v-flex>
-    </v-layout>
-    <v-layout primary row>
-      <v-spacer />
-      <v-btn @click.native="deleteSelectedFiles" flat><v-icon color="white" medium>delete</v-icon></v-btn>
-      <v-spacer />
-      <v-btn @click.native="downloadFiles" v-if="items.length <= 1" flat><v-icon color="white" medium>cloud_download</v-icon></v-btn>
-      <v-btn disabled v-else flat><v-icon color="white" medium>archive</v-icon></v-btn>
-      <v-spacer />
-    </v-layout>
-    <v-tabs
-      v-model="active"
-      color="primary lighten-5"
-      dark
-      slider-color="yellow"
-    >
-      <v-tab
-        v-for="tab of fileSideBars"
-        :key="tab.name"
-        ripple
+  <oc-app-side-bar>
+    <template slot="title">
```

Comment:
For my understanding: why are we going for this template/slot approach?

Vuetify uses explicit components which can be added here.

What I have in mind:

```html
<oc-app-side-bar>
 <oc-app-sidebar-title>{{ getTabName }} </oc-app-sidebar-title>
  <oc-side-bar-header>
   <v-layout>.....</v-layout>
  </oc-side-bar-header>
  <oc-side-bar-content>
    <v-layout>.....</v-layout>
  </oc-side-bar-content>
</oc-side-bar>
```

This is the cool stuff I like about vuetify: the compilation of components - this is what I'd like to have with our components as well.

Reply:
This can be done in a followup PR ;-)